src/io/result_io.jl

```diff
@@ -400,54 +400,51 @@ function setup_result_store!(domain::Domain, scen_spec::DataFrame)::Tuple
         return (dl...,)
     end
 
-    met_names = [
-        :relative_cover,
-        :relative_shelter_volume,
-        :absolute_shelter_volume,
-        :relative_juveniles,
-        :juvenile_indicator,
-        :coral_evenness
+    outcome_metrics::Vector{metrics.Metric} = [
```

Comment:
Does this need to be updated when a new metric is added? Is that documented?
Also, should it be at the top of the file or in some other file?

Reply:
I'm not sure how much you know about that so I'm sorry if I'm explaining something you already know: one thing is the `metrics` module and all metrics inside it; another thing are the outcomes that are saved in the ResultSet after the model is run. This vector you are pointing to has the metrics that correspond to these outcomes. 

About your questions: Everytime we decide to save a new outcome to the ResultSet this vector needs to be updated, but I don't think that this happened in the last year or so. All metrics within the `metrics` module are documented (some better than others). The set of outcomes saved in the ResultSet is not. This vector is only relevant in the context of this function that is building the `result_store` (the zarr that will hold this outcomes, among other things). 

I don't see a reason to move this to another file or to the top of this file, but I don't mind doing it. I don't think this needs to be documented, because in the end of the day what the user needs to know is all the metrics we have (and, again, these are documented). Some of these metrics just access these outcomes directly and that's how we expect the user to have access to them, and not by looking at `rs.outcomes[:relative_coral_cover]` direclty, for example. 

Let me know what do you think. :)

Reply:
Since someone adding a future resultset metric will probably be looking at this function anyway, then it's probably obvious.